dp/generate_climos.py

```diff
@@ -187,6 +189,15 @@ def create_climo_files(outdir, input_file, operation, t_start, t_end,
         'prsn': 'point'
     }
 
+    def generate_curr_time():
```

Comment:
We could use a more specific name for this function. `generate_curr_time` is pretty generic. An alternative would be something like `curr_time_cdo_format`.

A side note: For functions (which are side-effect free, as opposed to procedures, which exist primarily for their side effects), we often prefer to omit verbs like "generate" or "make" from their names unless doing so would be confusing. Those verbs are often generic and amount to "compute", which is already implicit in the notion of a function.